readthedocs/projects/tests/test_build_tasks.py

```diff
@@ -304,7 +304,7 @@ def test_build_updates_documentation_type(self, load_yaml_config):
         assert self.requests_mock.request_history[8]._request.method == "PATCH"
         assert self.requests_mock.request_history[8].path == "/api/v2/version/1/"
         assert self.requests_mock.request_history[8].json() == {
-            "addons": True,
+            "addons": False,
```

Comment:
We should not even send the `addons` fields on these API calls now.

Reply:
This is because we use a `APIVersion` model that inherits from the Version model, it will be removed when the field is removed from the model.